Student/Student/Submissions/SubmissionDetails/SubmissionDetailsPresenter.swift

```diff
@@ -64,10 +63,11 @@ class SubmissionDetailsPresenter {
 
     init(env: AppEnvironment = .shared, view: SubmissionDetailsViewProtocol, context: Context, assignmentID: String, userID: String) {
         self.context = context
-        self.assignmentID = assignmentID
-        self.userID = userID
+        self.assignmentID = ID.expandTildeID(assignmentID)
+        self.userID = ID.expandTildeID(userID)
```

Comment:
Do we need to start doing this everywhere? Should it be done in the router? Will it not affect querying CoreData by this new id?

Reply:
the problem was that the API request came back with expanded ids, but the htmlUrl has a tilde id, so it saved & looked up by different ids. Since we could deep link or route to tilde URLs we need to expand them at some point in routing.